src/App.css

```diff
@@ -1,42 +1,11 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap');
+
 #root {
   max-width: 1280px;
```

Comment:
If you want to add max-width, you also should add margin: 0 auto; --> it will center you content horizontally or screens larger than the max value